Camera2SlowMotion/app/src/main/java/com/example/android/camera2/slowmo/fragments/CameraFragment.kt

```diff
@@ -330,10 +330,10 @@ class CameraFragment : Fragment() {
 
                     // Starts recording animation
                     overlay.post(animationTask)
+
+                    // Send signal to start key simulation in server text editor.
```

Comment:
Rather than telling the server what to do, it should just be sending a notification of what state it is in,  ie this should be "tell the server recording has started".  This makes it conceptually easier to use the app in a different context, eg starting a window and detecting the menu text.

Reply:
Yeah that is implemented using ACTION codes trasmitted to server.